devlab/tests/config.py

```diff
@@ -458,8 +458,20 @@
           'name': 'segm_id_test_subnet_2', 'connect_to_ext_router': False,
           }
          ]
+     },
+    {'name': 'another_shared_net', 'admin_state_up': True, 'shared': True,
+     'provider:network_type': 'gre', 'provider:segmentation_id': 25,
+     'router:external': True, 'real_network': True,
+     'subnets': [
+         {'cidr': '192.168.55.0/24', 'ip_version': 4,
+          'name': 'another_ext_subnet', 'allocation_pools': [
+             {'start': '192.168.55.100', 'end': '192.168.55.254'}]
+          }]
      }]
 
+dst_unassociated_fip = 4
+"""Amount of unassociated Floating IP will be created on DST cloud."""
```

Comment:
Pleaseeee don't use the triple double quotes for comments. You can use it if you need a doc-string for that variable, but i'm not sure that python allows us to write the doc-strings for variables.